smother/cli.py

```diff
@@ -17,12 +17,22 @@
          'instead of individual line numbers.',
     is_flag=True,
 )
+@click.option(
+    '--coveragerc',
```

Comment:
let's call this 'rcfile' to [mirror coverage](https://coverage.readthedocs.io/en/coverage-4.2/config.html?highlight=coveragerc#configuration-files)